components/form-control/README.md

```diff
@@ -165,6 +165,27 @@ Clicking on the input with type `submit` will make a `GET` request to `http://lo
 * The data from `<input type="submit"/>` will be included in the request if this input is clicked.
 * The **name** attribute is required if you want your input to be included in the form data on submitting!
 
+## Select
+
+Native `<select>` element is not supported by Gameface. However, there is a polyfill that transforms `<select>` into a working dropdown inside the Gameface. That dropdown element - `<custom-select>` is supported by the form control component.
```

Comment:
1. in~side~ the Gameface Player
2. in~side~ Gameface

Reply:
Fixed